src/java/picard/vcf/LiftoverVcf.java

```diff
@@ -137,7 +138,7 @@ public static void main(final String[] args) {
         log.info("Lifting variants over and sorting.");
 
         final SortingCollection<VariantContext> sorter = SortingCollection.newInstance(VariantContext.class,
-                new VCFRecordCodec(outHeader),
+                new VCFRecordCodec(outHeader, VALIDATION_STRINGENCY != ValidationStringency.STRICT),
```

Comment:
The default is handled by `VariantContextWriterFactory`, which creates `VariantContextWriter`s, which internally uses a `VCFEncoder`.  The following was new to me when I was doing this - `VCFRecordCodec` is basically a custom wrapper around `VCFEncoder`, for `SortingCollection` to use so that it can write VCF records to files without having to write the header to each of them.  It didn't seem worth it to also add the ability to change the default behaviour of `VCFRecordCodec` .